profiles/commands.py

```diff
@@ -1,8 +1,12 @@
-from profiles.models import Name, Profile
+from iso3166 import countries
+from pendulum import create as date
+
+from profiles.models import Affiliation, Name, Profile
 
 
 def update_profile_from_orcid_record(profile: Profile, orcid_record: dict) -> None:
     _update_name_from_orcid_record(profile, orcid_record)
+    _update_affiliations_from_orcid_record(profile, orcid_record)
```

Comment:
these are starting to become potentially independent updates e.g. if it gets too slow we could do some of them in background

Reply:
They mostly will be, it's just the first authentication that requires it (we won't need to do it subsequent times as we'll be using the webhook).